drivers/hmis_csv_twenty_twenty_two/app/models/hmis_csv_twenty_twenty_two/exporter/enrollment_coc.rb

```diff
@@ -16,7 +16,7 @@ class EnrollmentCoc < GrdaWarehouse::Hud::EnrollmentCoc
     # HouseholdID is required, but often not provided, send some sane defaults
     # Also unique the HouseholdID to a data source
     def apply_overrides(row, data_source_id:)
-      row[:ProjectID] = project_id_from_enrollment_id(row[:EnrollmentID], data_source_id) if row[:ProjectID].blank?
+      row[:ProjectID] = project_id_from_enrollment_id(row[:EnrollmentID], data_source_id) # Force all EnrollmentCoC projects to be the same as their enrollment
```

Comment:
Maybe restrict this to `row[:DataCollectionState] == 1`? I think we decided that was the constraint required by the spec

Reply:
I believe ProjectID should always match Enrollment.ProjectID.  But CoCCode can differ.